src/wifi/WiFiManager.cpp

```diff
@@ -299,6 +300,20 @@ bool WiFiManager::Init() {
   WiFi.enableSTA(true);
   WiFi.setHostname(OPENSHOCK_FW_HOSTNAME);  // TODO: Add the device name to the hostname (retrieve from API and store in LittleFS)
 
+  wifi_config_t current_conf;
+  if (esp_wifi_get_config((wifi_interface_t)ESP_IF_WIFI_STA, &current_conf) == ESP_OK) {
+    char ssid[33];
+    char password[65];
```

Comment:
NitPick, but this should be 64 :p
https://stackoverflow.com/questions/18006390/why-is-the-wpa2-psk-key-length-limited-to-63-characters

Reply:
Also I don't see this being used anywhere, is it really needed?

Reply:
Oop, yeah, just force of habit to do both. Removing.